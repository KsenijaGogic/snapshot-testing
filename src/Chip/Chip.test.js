// We'll need to import React, as we're generating components

// We'll need to import a name helper called mount from 'enzyme'

// And we'll need to import a default helper called toJson from 'enzyme-to-json'


// We'll import our component here


// And we'll define our test here, using the global "it" helper
// The first argument is a string that will help is _describe_ our test
// The second argument is a function that the test will run

