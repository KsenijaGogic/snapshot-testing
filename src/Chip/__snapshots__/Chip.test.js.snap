// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should accept all props 1`] = `
<Chip
  code="Test-Chip-Code-1"
  hex="#B4D455"
  name="Badass Green"
>
  <div
    className="Chip"
  >
    <Card>
      <div
        className="Card"
      >
        <div
          className="Chip-header"
          style={
            Object {
              "backgroundColor": "#B4D455",
            }
          }
        />
        <div
          className="Chip-body"
        >
          <p
            className="Chip-tm"
          >
            Pantone®
          </p>
          <p
            className="Chip-code"
          >
            Test-Chip-Code-1
          </p>
          <p
            className="Chip-name"
          >
            Badass Green
          </p>
        </div>
      </div>
    </Card>
  </div>
</Chip>
`;

exports[`should render chip even with just hex code passed 1`] = `
<Chip
  hex="#B4D455"
>
  <div
    className="Chip"
  >
    <Card>
      <div
        className="Card"
      >
        <div
          className="Chip-header"
          style={
            Object {
              "backgroundColor": "#B4D455",
            }
          }
        />
        <div
          className="Chip-body"
        >
          <p
            className="Chip-tm"
          >
            Pantone®
          </p>
          <p
            className="Chip-code"
          >
            Please specify code
          </p>
          <p
            className="Chip-name"
          >
            Please specify name
          </p>
        </div>
      </div>
    </Card>
  </div>
</Chip>
`;
